packages/foreman/foreman-bootloaders-redhat/foreman-bootloaders-redhat.spec

```
@@ -11,7 +11,9 @@ BuildArch: noarch
 Source0: foreman-generate-bootloaders
 
 Requires: grub2-tools-extra
+%if 0%{?rhel} == 0
```

Should this be:
```suggestion
%if 0%{?rhel} < 9
```

On Fedora it's still available and there it would evaluate to `0 < 9` which is true. So technically correct, even if it's irrelevant for us.